Dockerfile.ppc64le.ubi

```diff
@@ -1,19 +1,16 @@
-ARG BASE_UBI_IMAGE_TAG=9.5-1739420147
-
+ARG BASE_UBI_MIN_IMAGE_TAG=9.5-1741599792
```

Comment:
Can we use only one? `BASE_UBI_IMAGE_TAG` seems enough, or is there a reason why you defined two?